src/BlogPostPage.jsx

```diff
@@ -87,20 +298,183 @@ if (blog) {
     return <Widget src="${REPL_ACCOUNT}/widget/Posts.ModeratedPostData" props={{ ...dataProps, renderPost }} />;
   };
 
+  const postUrl = `https://near.org/near/widget/BlogPostPage?accountId=${props.accountId}&blockHeight=${props.blockHeight}`;
+  const markdownObj = parseMarkdown(blog.blogContent);
+
+  const renderComment = (a) => {
+    return (
+      <div key={JSON.stringify(a)}>
+        <Widget
+          src={`${REPL_ACCOUNT}/widget/Comments.Comment`}
+          props={{
+            accountId: a.account_id,
+            blockHeight: a.block_height,
+            content: a.content,
+            highlight:
+              a.account_id === props.highlightComment?.accountId &&
+              a.block_height === props.highlightComment?.blockHeight,
+            GRAPHQL_ENDPOINT,
+            notifyAccountId,
+            item,
+          }}
+        />
+      </div>
+    );
+  };
+
+  const renderedComments = blog.blogComments?.map(renderComment);
+
   return (
     <>
-      <Widget
-        src="${REPL_ACCOUNT}/widget/DIG.Button"
-        props={{
-          label: "Back",
-          href: `/${REPL_ACCOUNT}/widget/Blog.Feed`,
-          iconLeft: "ph ph-arrow-left",
-          variant: "secondary",
-          size: "small",
-        }}
-        style={{ marginBottom: "2em" }}
-      />
-      <Widget src="${REPL_ACCOUNT}/widget/Moderation.CheckPostModeration" props={{ ...props, renderData }} />;
+      <BlogPostWrapper>
+        <Widget
+          src="${REPL_ACCOUNT}/widget/DIG.Button"
+          props={{
+            label: "Back To All Posts",
+            href: `/${REPL_ACCOUNT}/widget/Blog.Feed`,
+            iconLeft: "ph ph-arrow-left",
+            variant: "secondary",
+            size: "small",
+          }}
+          style={{ marginBottom: "1rem" }}
+        />
+        {/* RENDER BLOG HEADER IMAGE */}
+        <PostImage
+          imageUrl={
+            markdownObj[0].imageUrl || "https://pages.near.org/wp-content/uploads/2023/06/generic-green-blog.png"
```

Comment:
Nice default. This is fine for now, but please track moving this to ipfs so we do not increase our dependency on pages.near.org.

Reply:
moved to ipfs